src/main/java/org/fogbowcloud/saps/engine/core/archiver/storage/nfs/NfsPermanentStorage.java

```diff
@@ -20,31 +26,41 @@
 public class NfsPermanentStorage implements PermanentStorage {
 
     public static final Logger LOGGER = Logger.getLogger(NfsPermanentStorage.class);
-    static final String NFS_STORAGE_TASK_DIR_PATTERN = "%s" + File.separator + "%s" + File.separator + "%s";
+    public static final String NFS_STORAGE_TASK_DIR_PATTERN = "%s" + File.separator + "%s" + File.separator + "%s";
+    public static final String NFS_STORAGE_TASK_DATA_URL_PATTERN = "%s" + File.separator + "%s";
```

Comment:
Change privacy to private.

Reply:
Rename `NFS_STORAGE_TASK_DATA_URL_PATTERN` to `NFS_STORAGE_TASK_URL_PATTERN`

Reply:
@wesleymonte 
I renamed this pattern, but I cannot change it to `private`, because it is used in tests. If I modify it I will have to duplicate it in the test to use, what do you think?

Reply:
Resolved with @wesleymonte